code/screens/courtrecords.lua

```diff
@@ -1,81 +1,83 @@
 function DrawCourtRecords(ui)
     local bodyOptions = {}
+    local bodySelected = nil
+    local menuButtons = {
+        {
+            title = "Back",
+            key = controls.press_court_record
+        },
+        -- TODO: Can you present a profile?
```

Comment:
From the wiki,
> In most games, profiles may not be presented unless forced to do so. However, in Phoenix Wright: Ace Attorney: Justice For All and Phoenix Wright: Ace Attorney: Trials and Tribulations, profiles can be presented in the same way as evidence in both investigations and trials.

So I guess it will depend on what the writers need.